modules/persistent-storage-csi-group-snapshots-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-group-snapshots-limitations_{context}"]
+= CSI volume group snapshots limitations
+
+Volume group snapshots has the following limitations:
+
+* Does not support reverting an existing persistent volume claim (PVC) to an earlier state represented by a snapshot (only supports provisioning a new volume from a snapshot).
```

Comment:
Per ISG: "Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text."
Feel free to modify the sentence. My suggestion is just an example.

```suggestion
* Does not support reverting an existing persistent volume claim (PVC) to an earlier state represented by a snapshot, and supports provisioning a new volume from a snapshot.
```